packages/components/package.json

```diff
@@ -55,6 +56,7 @@
     "ember-stargate": "^0.4.3",
     "ember-style-modifier": "^4.4.0",
     "ember-truth-helpers": "^4.0.3",
+    "luxon": "^2.3.2 || ^3.4.2",
```

Comment:
I suggest we add `luxon` (2.x or 3.x) and `ember-concurrency` (4.x) to the changelog entry

Reply:
Updated. Does it look ok the way I formatted it?